Bus Scheduler/src/components/tripSearch/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { useFetchTrips } from './store/hook';
+
```

Comment:
if this component is part of a page then  /features/home/components/tripSearch or else if it is shared /src/shared/components/tripSearch/

Reply:
Done